docs/scarpet/api/Entities.md

```diff
@@ -670,6 +678,18 @@ if the custom name is always visible, even through blocks.
 Sets the entity persistence tag to `true` (default) or `false`. Only affects mobs. Persistent mobs
 don't despawn and don't count towards the mobcap.
 
+### `modify(e, 'item', item_triple)`
+
+Sets the item for the item or item frame entity.
```

Comment:
You could mention something like (The item triple is a list of `[item_name, count, nbt]`